modules/auxiliary/server/ldap_server.rb

```diff
@@ -0,0 +1,89 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::LDAP::Server
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Native LDAP Server (Example)',
+      'Description'    => %q{
+        This module provides a Rex based LDAP service to expose the
+        native Rex LDAP server functionality created during log4shell
+        development.
+      },
+      'Author'         => 'RageLtMan <rageltman[at]sempervictus>',
+      'License'        => MSF_LICENSE,
+      'References'     => []
```

Comment:
Should this not be setup as a passive service by default?
```suggestion
     'References'     => [],
     'Actions' =>
       [
         [ 'Service', 'Description' => 'Run LDAP server' ]
       ],
     'PassiveActions' =>
       [
         'Service'
       ],
     'DefaultAction' => 'Service'
```

Reply:
yeah probably, and needs all the references and docs and stuff, plus prolly put you at least into the authors page - uses a chunk of logic you came up with during your initial impl.

Reply:
done